packages/wdio-browserstack-service/src/accessibility-handler.ts

```diff
@@ -176,22 +184,15 @@ class _AccessibilityHandler {
 
             if (shouldScanTestForAccessibility) {
                 BStackLogger.info('Automate test case execution has ended. Processing for accessibility testing is underway. ')
-            }
 
-            const dataForExtension = {
-                saveResults: shouldScanTestForAccessibility,
-                testDetails: {
-                    'name': test.title,
-                    'testRunId': process.env.BS_A11Y_TEST_RUN_ID,
-                    'filePath': this._suiteFile,
-                    'scopeList': [suiteTitle, test.title]
-                },
-                platform: this._platformA11yMeta
-            }
+                const dataForExtension = {
+                    'thTestRunUuid': process.env.TEST_ANALYTICS_ID,
+                    'thBuildUuid': process.env.BROWSERSTACK_TESTHUB_UUID,
+                    'thJwtToken': process.env.BROWSERSTACK_TESTHUB_JWT
+                }
 
-            await this.sendTestStopEvent((this._browser as WebdriverIO.Browser), dataForExtension)
+                await this.sendTestStopEvent((this._browser as WebdriverIO.Browser), dataForExtension)
```

Comment:
unnecessary leading space

Reply:
This statement was moved inside an if block hence the indendation.